src/vs/workbench/contrib/search/browser/anythingQuickAccess.ts

```diff
@@ -71,6 +71,46 @@ function isEditorSymbolQuickPickItem(pick?: IAnythingQuickPickItem): pick is IEd
 	return !!candidate?.range && !!candidate.resource;
 }
 
+export class EditorViewState {
```

Comment:
I don't think this should live here. I don't think TextSearchQA should depend on anything in AnythingQA because AnythingQA is more of a gluing of other QA providers together into one "everything" experience

Reply:
Where do you think this should live?

Reply:
I agree, maybe in `src/vs/platform/quickinput/browser/quickAccess.ts`? Or even `src/vs/workbench/browser/quickaccess.ts` if only used in workbench.